README.md

```diff
@@ -1,44 +1,114 @@
 # filebrowser-upload
 
-Upload command for https://github.com/filebrowser/filebrowser
+Upload command for <https://github.com/filebrowser/filebrowser>
 
 ## Install
 
-Download binary executable from https://github.com/guyskk/filebrowser-upload/releases
+Download binary executable from <https://github.com/guyskk/filebrowser-upload/releases> or build with
+
+```console
+foo@bar:~$ ./release.sh
+```
+
+In ```dist``` folder you will find the executable.
 
 ## Usage
 
-```
-usage: filebrowser-upload [-h] [--version VERSION] --api API --username
-                          USERNAME --password PASSWORD --dest DEST
-                          [--insecure] [--no-progress] [--override]
-                          filepath
+```console
+usage: filebrowser-upload [-h] [--version VERSION] --api API --username USERNAME 
+                          [--password PASSWORD] [--insecure] [--no-progress] 
+                          [--override] [--dry-run]
+                          {file,folder} ...
 
 Filebrowser upload.
 
 positional arguments:
-  filepath             local file path
+  {file,folder}        Commands for single file upload or entire folder
+    file               Single file upload
+    folder             Entire folder and subfolders upload
 
 optional arguments:
   -h, --help           show this help message and exit
   --version VERSION    Show version
   --api API            Filebrowser upload API URL
   --username USERNAME  Username
-  --password PASSWORD  Password
-  --dest DEST          File destination
+  --password PASSWORD  Inline password
   --insecure           Allow insecure server connections when using SSL
   --no-progress        Disable progress bar
-  --override           Override file or not
+  --override           Override files or not
+  --dry-run            Dry run mode (no upload)
+```
+
+### Single file help
+
+```console
+usage: filebrowser-upload file [-h] src dest
```

Comment:
what's `file` means? the program should auto detect file or folder.

Reply:
Introduced auto detect